src/app/public/modules/list-view-grid/list-view-grid.component.ts

```diff
@@ -232,13 +258,38 @@ export class SkyListViewGridComponent
     this.gridDispatcher.next(new ListViewGridColumnsLoadAction(columnModels, true));
 
     this.handleColumnChange();
+
+    if (this.enableMultiselect) {
+      this.dispatcher.toolbarShowMultiselectToolbar(true);
+    }
   }
 
   public ngOnDestroy() {
     this.ngUnsubscribe.next();
     this.ngUnsubscribe.complete();
   }
 
+  public multiselectSelectionChange(event: any): void {
```

Comment:
Is this literally an `any` type, or is there a specific type that could be injected here?

Reply:
nope, its `SkyGridSelectedRowsModelChange`. fixed.